gmso/formats/gro.py

```diff
@@ -57,31 +57,30 @@ def read_gro(filename):
         coords = u.nm * np.zeros(shape=(n_atoms, 3))
         for row, _ in enumerate(coords):
             line = gro_file.readline()
-            content = line.split()
             if not line:
                 msg = (
                     "Incorrect number of lines in .gro file. Based on the "
                     "number in the second line of the file, {} rows of"
                     "atoms were expected, but at least one fewer was found."
                 )
                 raise ValueError(msg.format(n_atoms))
+            res_id = int(line[:5].strip())
+            res_name = line[5:10].strip()
+            atom_name = line[10:15].strip()
+            atom_id = line[15:20].strip()
 
-            res = content[0]
-            atom_name = content[1]
-            atom_id = content[2]
+            positions = line[20:].split()
```

Comment:
The positions are still fixed width.  I think it would likely be better to use explicit character locations to be completely consistent with the original file format. Granted, this should produce the same exact result, since there is always a space between each component of the position, but I think better to follow the explicit documentation. 

I think it would be good to add a comment in the code here as well, with the file format info:

> residue number (5 positions, integer)
> residue name (5 characters)
> atom name (5 characters)
> atom number (5 positions, integer)
> position (in nm, x y z in 3 columns, each 8 positions with 3 decimal places)
> velocity (in nm/ps (or km/s), x y z in 3 columns, each 8 positions with 4 decimal places)
> 
> "%5d%-5s%5s%5d%8.3f%8.3f%8.3f%8.4f%8.4f%8.4f"

Might also want to update doctoring to note that velocity is ignored if not provided in the file.

Reply:
Yeah, I think I will need to double check the case with positions, since it seems like their could be both the scenario where there's only 3 decimal and full 8 decimals, so my fear is having fixed width will only work for either case and not both. (like the example in https://manual.gromacs.org/archive/5.0.3/online/gro.html is having the shorter version).
I will add a note about velocity being ignored.